pontoon/contributors/static/js/profile.js

```diff
@@ -0,0 +1,137 @@
+var Pontoon = (function (my) {
+  return $.extend(true, my, {
+    insights: {
+      renderCharts: function () {
```

Comment:
```suggestion
      renderCharts() {
```

Prettier with this sugar. Same for later in the file as well.

Reply:
I'd like to avoid adding more inconsistencies to our code style, so let's do this at a later point across the entire module.